src/views/GetStarted/CoachingStyle/index.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { menApiAuthClient } from "../../../clients/mentumm";
+import { StyleType } from "../../../types";
+import {
+  Button,
+  Checkbox,
+  Container,
+  Flex,
+  Icon,
+  Heading,
+  Stack,
+  Text,
+  Spinner,
+} from "@chakra-ui/react";
+import {
+  FaRocket,
+  FaHatWizard,
+  FaSeedling,
+  FaAnchor,
+  FaLightbulb,
+  FaFire,
+  FaChessKing,
+} from "react-icons/fa";
+import envConfig from "../../../envConfig";
+
+type StyleTypeOptionProps = {
+  styleType: StyleType;
+  checkedItems: String[];
+  setCheckedItems: Function;
+};
+
+const getStyleTypeIcon = (icon) => {
+  switch (icon) {
```

Comment:
slick way to render icons, when we iterate on this page, the db table also stores colors for these styles